File: SDKs/amazonaws.com/sesv2/2019-09-27/python/src/sdk/models/shared/body.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import content as shared_content


@dataclass_json
@dataclasses.dataclass
class Body:
    r"""Body
    Represents the body of the email message.
    """
    
    html: Optional[shared_content.Content] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Html') }})
    text: Optional[shared_content.Content] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Text') }})
    

File: SDKs/amazonaws.com/sesv2/2019-09-27/java/lib/src/main/java/openapisdk/models/shared/Body.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * Body
 * Represents the body of the email message.
**/
public class Body {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Html")
    public Content html;
    public Body withHtml(Content html) {
        this.html = html;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Text")
    public Content text;
    public Body withText(Content text) {
        this.text = text;
        return this;
    }
}